hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ReconBasicOmKeyInfo.java

```diff
@@ -247,6 +247,30 @@ public static ReconBasicOmKeyInfo getFromProtobuf(OzoneManagerProtocolProtos.Key
     return builder.build();
   }
 
+  public static ReconBasicOmKeyInfo getFromProtobuf(OzoneManagerProtocolProtos.KeyInfo keyInfoProto) {
```

Comment:
Any reason you duplicated this method ?

Reply:
Yes, the existing method takes `OzoneManagerProtocolProtos.KeyInfoProtoLight` as an argument while the new overloaded method takes `OzoneManagerProtocolProtos.KeyInfo` as an argument!